tests/objects/test_ontology.py

```diff
@@ -141,7 +158,6 @@ def test_json_to_ontology():
 
     # WHEN
     actual = encord.objects.OntologyStructure.from_dict(editor_dict)
-
```

Comment:
What's your beef with the empty lines? they are rather useful!

Reply:
I think these are all places where I have hacked around and then not resolved the diff!